pyproject.toml

```diff
@@ -29,11 +29,11 @@ black = "^22.12.0"
 isort = "^5.11.4"
 
 [tool.poetry.group.lint.dependencies]
-flake8 = "^5.0.4"
+flake8 = "^6.0.0"
 flake8-docstrings = "^1.7.0"
-flake8-copyright = "^0.2.3"
+flake8-copyright = "^0.2.4"
 flake8-builtins = "^2.1.0"
-pyproject-flake8 = "^5.0.4.post1"
+pyproject-flake8 = "^6.0.0.post1"
```

Comment:
`flake8` 6 seems to work now.

Reply:
Yeap, https://github.com/savoirfairelinux/flake8-copyright/issues/19 is closed.
Can you please remove `flake8` pinning from PSQL repos.
I have sent a MM notification to the team to clean pinning from their repos.

Nice!